<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fifteen</class>
 <widget class="QMainWindow" name="Fifteen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fifteen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 255, 200);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>295</width>
      <height>295</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn10">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn13">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>13</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn5">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn15">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn14">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>14</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn8">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn11">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>11</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn7">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn6">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn9">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btn12">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>12</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn4">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="controlBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>160</width>
      <height>260</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>140</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>260</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>150</width>
       <height>184</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_shufle">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Shuffle</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>7</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Button size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="btnSizeBox">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Layout size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="layoutSizeBox">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="count_move">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Count move</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="count">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QComboBox" name="levelBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Clasical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Palm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rome</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Castle</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btnSort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 100, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Sort</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
